lib/LTP/utils.pm

```diff
@@ -46,11 +47,21 @@ sub shutdown_ltp {
     loadtest_kernel('shutdown_ltp', @_);
 }
 
+sub get_ltproot {
+    my $test;
+    my $prefix = '/opt/ltp';
+
+    get_required_var('TEST');
+    $prefix .= '-32' if (m/[-_]m32$/);
+
+    return $prefix;
+}
```

Comment:
```
return get_required_var('TEST') =~ m/[-_]m32$/ ? '/opt/ltp-32' : '/opt/ltp'
```

Do I get it right, that test suite name, you should determine what lptroot to use?

Reply:
Sure, it can be one liner, ok I replace it :).
Yes, *both* `LTPROOT` environment variable and `--prefix` parameter to configuration script refer to LTP installation directory. Feel free to suggest best name for function :).

Reply:
Maybe `get_ltp_install_directory` would be more readable for people outside LTP community.

Reply:
I am completely fine, no worries! I just tend to forget about magical variable `$_` in perl. :)

Reply:
Thanks!